integration/spec/features/v2/save_and_return_spec.rb

```diff
@@ -19,30 +21,66 @@
     form.start_now_button.click
 
     check_optional_text(page.text)
-    form.question_1.set('hello')
+    form.question_1.set(q1_answer)
     continue
     form.save_and_return.click
     form.email.set('email@')
     continue
     check_validation_error_message('Enter an email address in the correct format, like name@example.com')
-    form.email.set('email@example.com')
+    form.email.set('fb-acceptance-tests@digital.justice.gov.uk')
     continue
-    check_validation_error_message("can't be blank")
+    check_validation_error_message('Enter an answer for "Secret question"')
     form.secret_question_1.choose
     form.secret_question_2.choose
     form.secret_question_3.choose
```

Comment:
It's implicitly tested because it proceeds to the next page but I like the explicit expectation to prevent regression as it does a bit of weirdness to ensure it keeps the value of the selection and the text for later (while also being able to change the questions without invalidating old saved progress....). Added 👍